superset/views/core.py

```diff
@@ -2594,16 +2594,16 @@ def validate_sql_json(self):
             return json_error_response(f"{msg}")
 
     @has_access_api
-    @expose("/sql_json/", methods=["POST", "GET"])
+    @expose("/sql_json/", methods=["POST"])
```

Comment:
were we allowing `GET` for some caching reason? @betodealmeida might have context

Reply:
Nvm, i think this is safe, and this request should always be a `POST` regardless since you might be altering data